server/src/Controller/Message/MessageController.js

```diff
@@ -0,0 +1,103 @@
+import {
+  create,
+  deleteDoc,
+  findOne,
+  findMany,
+  update,
+} from '../../Repositories/genericRepository';
+import successHandler from '../../libs/routes/successHandler';
+import { messageModel } from '../../Model';
+
+
+const getMessage = async (req, res, next) => {
+  try {
+    const { id } = req.params;
+    const result = await findOne(messageModel, id);
+    if (!result) {
+      const err = new Error('Message does not exists');
+      next(err);
+    }
+    res
+      .status(200)
+      .send(successHandler('Message is here', result));
+  } catch (error) {
+    next(error);
+  }
+};
+
+const getAllMessages = async (req, res, next) => {
+  try {
+    const result = await findMany(messageModel);
```

Comment:
I think we will never use the get all messages API. We can call API using timestamp though, like get all messages before the timestamp T.